zebra-consensus/src/error.rs

```diff
@@ -56,6 +57,13 @@ pub enum TransactionError {
     #[error("coinbase inputs MUST NOT exist in mempool")]
     CoinbaseInMempool,
 
+    #[error("transaction is locked until block height {}", _0.0)]
+    LockedUntilBlockHeight(block::Height),
+
+    #[error("transaction is locked until {0}")]
+    #[cfg_attr(any(test, feature = "proptest-impl"), proptest(skip))]
+    LockedUntilDateTime(DateTime<Utc>),
```

Comment:
Required Consensus Rule Clarifications:

1. this is a block time, not a user's clock time
2. the lock still applies if the block time is equal to this time

```suggestion
    #[error("transaction is locked until after block time {0}")]
    #[cfg_attr(any(test, feature = "proptest-impl"), proptest(skip))]
    LockedUntilAfterDateTime(DateTime<Utc>),
```

Reply:
Done in [b15fb1e](https://github.com/jvff/zebra/commit/b15fb1efcd86611316d48d1c8d55a8f7a0f826c0) and the new names are used in [85df78b](https://github.com/jvff/zebra/commit/85df78b8d3c3de508460f89dfa36d6833467a6a2).